### ISSUE TITLE
False positive: Property 'snapshot' does not exist on type '{}' for component exports

### ISSUE THREAD

**Issue by janosh** [pheuter/svelte-check-rs#89]
When a Svelte 5 component exports a `snapshot` object (for SvelteKit snapshot support) and you try to access it from a mounted component instance in tests, svelte-check-rs types the component instance as `{}` instead of including the exported members.

## Reproduction

**Component (repro-snapshot-export.svelte):**
```svelte
<script lang="ts">
  let count = $state(0)

  export const snapshot = {
    capture: () => ({ count }),
    restore: (values: { count: number }) => (count = values.count),
  }
</script>

<button onclick={() => count++}>Count: {count}</button>
```

**Test file (repro-snapshot-test.svelte.ts):**
```ts
import Component from './repro-snapshot-export.svelte'
import { mount, tick } from 'svelte'
import { describe, expect, it } from 'vitest'

describe(`snapshot export`, () => {
  it(`should access snapshot from mounted component`, async () => {
    const component = mount(Component, {
      target: document.body,
      props: {},
    })

    // This triggers the error
    expect(component.snapshot.capture()).toEqual({ count: 0 })
    component.snapshot.restore({ count: 5 })
  })
})
```

## Error

```
Error: Property 'snapshot' does not exist on type '{}'. (ts(TS2339))
```

svelte-check-rs: 0.7.1
